test/e2e/controllers.go

```diff
@@ -26,9 +26,9 @@ const (
 // knownControllers contains all controllers to be registered when starting the e2e test suite
 var knownControllers = map[string]func(controllerName string, mgr ctrl.Manager) error{
 	// Organization controllers.
-	"organizationNamespace":      (&organizationcontrollers.NamespaceReconciler{}).SetupWithManager,
-	"organizationRBAC":           (&organizationcontrollers.RBACReconciler{}).SetupWithManager,
-	"organizationDEX":            startOrganizationDexReconciler,
+	"organizationNamespace": (&organizationcontrollers.NamespaceReconciler{}).SetupWithManager,
+	"organizationRBAC":      (&organizationcontrollers.RBACReconciler{}).SetupWithManager,
+	//"organizationDEX":            startOrganizationDexReconciler, //Temporarily commented because it cannot deploy the Plugin
```

Comment:
@uwe-mayer could we add applying the Dex CRDs also if `test.StartControlPlane(..)` is called with installCRDs=true?

Reply:
Yes, we should be doing this.
Kind installation [already does so](https://github.com/cloudoperators/greenhouse/blob/6626edc5f7ba2c9002ae044cc39ab425122c1e6f/test/e2e/local-cluster/create_cluster.go#L177)...
We should add the relevant code to `envtest` [here](https://github.com/cloudoperators/greenhouse/blob/main/pkg/test/env.go#L228)

Reply:
@gciezkowski-acc could you add the path to the CRDs of the idproxy chart there? Then we do not need to comment the DEX controller anymore.

Reply:
Done